assets/queries/terraform/azure/small_activity_log_retention_period/query.rego

```diff
@@ -0,0 +1,47 @@
+package Cx
+
+import data.generic.common as commonLib
+
+CxPolicy[result] {
+	monitor := input.document[i].resource.azurerm_monitor_log_profile[name]
+
+	object.get(monitor, "retention_policy", "undefined") == "undefined"
```

Comment:
It is unnecessary to check the existence of "retention_policy" because according to the documentation it is required. However, "retention_policy.days" is optional, so check that instead.

```suggestion
	object.get(monitor.retention_policy, "days", "undefined") == "undefined"
```